plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/TableStatisticsMaker.java

```diff
@@ -63,13 +63,17 @@ public static TableStatistics getTableStatistics(TypeManager typeManager, Iceber
     private TableStatistics makeTableStatistics(IcebergTableHandle tableHandle)
     {
         if (tableHandle.getSnapshotId().isEmpty()) {
-            return TableStatistics.empty();
+            return TableStatistics.builder()
```

Comment:
nit: a constant for `TableStatistics.builder()
                    .setRowCount(Estimate.of(0))
                    .build();`?

Reply:
i would call it "empty_table_statisics" but then it would look the same as `TableStatistics.empty();`
maybe we should rename `TableStatistics.empty()` in a follow-up?